.gitlab-ci.yml

```diff
@@ -1,12 +1,12 @@
-# Disable git shallow clone. We need full history for validating copyright year of each file.
-git:
-    depth: false
 
-env:
-    global:
-        TEST_BRANCH: $TRAVIS_BRANCH
+stages:
+  - test
 
-before_script:
+test:unit:
```

Comment:
There are no unit tests as such here, I suggest we rename the job to `test:changelog-generator` or something generic like `test:extra-tools`

Reply:
Done, using `test:extra-tools`